src/tests/redux/reducers/userReducer.spec.js

```diff
@@ -0,0 +1,33 @@
+import userReducer from '../../../redux/reducers/userReducer';
+
+describe('Test userReducers', () => {
+  const data = {
+    username: 'username',
+    email: 'mail',
+    image: 'image',
+  };
+  // const initialState = {
```

Comment:
Can you please remove these commented codes here.

Reply:
Thanks for pointing this out, It has been removed